### ISSUE TITLE
Library builds bad urls for olingo v4

### ISSUE THREAD

**Issue by ieugen** [janhommes/o.js#36]
Hello,

I'm using this library to request data from a java server with olingo v4

```
o().config({
  endpoint: '/api'
})

o('ShopWebsites').get(function (data) {
  console.log(`Data is {data}`)
})
```

I get this 

```
o.js:1384 GET http://localhost:3000/api/ShopWebsites/?$format=json 400 (Bad Request)
```

The request is bad because it adds the last slash after `ShopWebsites` .  If I make the request like: `/api/ShopWebsites?$format=json` it works.

**Comment by ieugen** [comment]
This fixes the request: 

https://github.com/netdava/o.js/commit/55a0038b6e8b9bffc6f9a5262bbe3d5e76547559

**Comment by johnsmith9264** [comment]
maybe make a pull request?

**Comment by janhommes** [comment]
Closed with [v0.3.1](https://github.com/janhommes/o.js/releases/tag/v0.3.1)
